catalog/catalog.proto

```diff
@@ -1,5 +1,6 @@
 syntax = "proto3";
 option go_package = "github.com/treevese/lakefs/catalog";
+option java_package = "io.treeverse.catalog";
```

Comment:
```suggestion
option java_package = "io.treeverse.lakefs.catalog";
```
One day Treeverse will be a huge multinational conglomerate.  It will have many divisions, among them lakeFS, Facebook, Google, and AWS.  Let's not waste the previous `io.treeverse` namespace!